android/src/main/java/com/expensify/livemarkdown/MarkdownUtils.java

```diff
@@ -164,8 +164,6 @@ private void applyRange(SpannableStringBuilder ssb, String type, int start, int
           depth);
         setSpan(ssb, span, start, end);
         break;
-      default:
-        throw new IllegalStateException("Unsupported type: " + type);
```

Comment:
I don't exactly know the use case, but wouldn't replacing this with a warn be less invasive? In case something unexpected ends up here it would still leave a trace. But maybe silently skipping unexpected things really is better, WDYT?

Reply:
Let's just silently ignore things for now (like we do on iOS) and address this idea in a separate PR.